core/src/Serial/Kokkos_Serial.hpp

```diff
@@ -52,6 +52,9 @@ class SerialInternal {
  public:
   SerialInternal() = default;
 
+  SerialInternal(const SerialInternal&) = delete;
```

Comment:
Why just these two instead of the Rule-of-5?

Technically this is just for emphasis, as having a `mutex` as a non-static member will delete all the move/copy operations.

Reply:
> Technically this is just for emphasis, as having a mutex as a non-static member will delete all the move/copy operations.

Correct. I'm happy to remove if you prefer.

Reply:
Let's remove it.

Reply:
> Let's remove it.

Done.